dev/grid.html

```diff
@@ -28,14 +28,26 @@
 
         cb(pageItems, levelSize);
       };
+
```

Comment:
Let's exclude these dev page changes. I added them in the quick proto but didn't mean for them to end up in master.